src/components/MenuItemCard.tsx

```diff
@@ -7,17 +7,19 @@ export type MenuCardProps = {
   icon: JSX.Element;
   label: string;
   textColor?: 'default' | 'red' | 'blue';
+  testId?: string;
   onClick?: () => void;
 };
 
 export const MenuItemCard: FunctionComponent<MenuCardProps> = ({
   icon,
   label,
   textColor = 'default',
+  testId = 'menu-item-card',
```

Comment:
I prefer not sending any test ids as props. I understand it might be required to test multiple menucard items. May be label can be used as suffix or prefix to identify it.

Reply:
The only problem here is, `label` here will be point to  <Text> field in this component and it is not clickable.

Reply:
in this point, data-testid is more straight way which have already been used in many components in the system. i will change the code a little bit to not pass the testId from parent, but directly use label way with `-` in the middle to hider the test id in component attribute.

Reply:
Yes, I meant the same. e.g data-testid = 'menu-item-card-'{label}